test/read_mixin_test.rb

```diff
@@ -128,6 +128,12 @@
       end
     end
 
+    it "finds cached when searching by column" do
+      Post.where(title: 'no_comments').first.title.must_equal 'no_comments'
+      Post.expects(:find_by_sql_without_kasket).never
+      Post.where(title: 'no_comments').first.title.must_equal 'no_comments'
```

Comment:
this is not testing ordering logic ... just saying "when search for an indexed column it should use the cache" ... how/why that works is not in scope ... that test would belong in the visitor test